src/common/paths/buckets_bucketID_labels.yml

```diff
@@ -29,31 +57,100 @@ post:
   tags:
     - Buckets
   summary: Add a label to a bucket
+  description: |
+    Adds a label to a bucket and returns the new label information.
+
+    Labels are objects that contain `labelID`, `name`, `description`, and `color`
```

Comment:
```suggestion
    Use this endpoint to add a label to a bucket.
    Labels are objects that contain `labelID`, `name`, `description`, and `color`
```

Reply:
going to skip this since we also skipped it for `list all labels` because it sounded redundant.